### PR TITLE
added onBidWon event

### PR DESCRIPTION
**Pull request by adxpremium**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
onBidWon event

### CODE REVIEW

**Review by robertrmartinez — CHANGES_REQUESTED** [review]
Looks fine, but please add a unit test.

**Comment by adxpremium** [comment]
> Looks fine, but please add a unit test.

Done :)

File: test/spec/modules/luponmediaBidAdapter_spec.js [hunk 1]
```diff
@@ -364,4 +364,34 @@ describe('luponmediaBidAdapter', function () {
...
+      'responseTimestamp': 1598527728026,
+      'requestTimestamp': 1598527727629,
+      'bidder': 'luponmedia',
+      'adUnitCode': 'div-gpt-ad-1533155193780-5',
+      'timeToRespond': 397,
+      'size': '300x250',
+      'status': 'rendered'
+    };
+
+    spec.onBidWon(bids);
```
Comment (robertrmartinez) [inline test/spec/modules/luponmediaBidAdapter_spec.js:395]: This does not actually test anything.

See this https://github.com/prebid/Prebid.js/blob/master/test/spec/modules/adpartnerBidAdapter_spec.js#L199 example.

From Line 199 and on.

You can see they trigger the `spec.onBidWon` but are validating the return, as well as stubbing the request and making sure it has the right data they expect.

**Review by robertrmartinez — CHANGES_REQUESTED** [review]

File: test/spec/modules/luponmediaBidAdapter_spec.js (hunk 1 above)
Comment (adxpremium) [reply test/spec/modules/luponmediaBidAdapter_spec.js]: Hi @robertrmartinez we just made the change that you suggested.

**Comment by robertrmartinez** [comment]
Please remove / revert the package-lock.json file from the staged commits and we can merge this

**Comment by adxpremium** [comment]
> Please remove / revert the package-lock.json file from the staged commits and we can merge this

@robertrmartinez Done

**Review by robertrmartinez — APPROVED** [review]

**Pull request merged**